tests/unit/browser/test_adblock.py

```diff
@@ -0,0 +1,434 @@
+# TODO See utils/test_standarddirutils for OSEError and caplog assertion
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+#!/usr/bin/env python3
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Comment:
That should probably be your name and 2015 only - not that I care much, but let's be correct :wink: